tests/extensions/default/analyzers/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers.Test/assets/TestClasses/UA0008.Fixed.cs

```diff
@@ -7,7 +7,7 @@ public class UA0008 : IUrlHelper
     {
         public IUrlHelper Method1(this IUrlHelper h)
         {
-            var x = h;
+            IUrlHelper x = h;
```

Comment:
this looks like a behavioral change. is this intentional?

Reply:
Yes, it's a result of me not applying the reducer now which I think is good because it keeps the var/not-var details the same as before the code fix was applied.